### PR TITLE
Created homework_5

### PR DESCRIPTION
**Pull request by Doctor-Ledenec**

### CODE REVIEW

File: src/homework_5/ResizableCircle.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package homework_5;
+
+import java.io.IOException;
+import java.nio.CharBuffer;
+
+
+public class ResizableCircle extends Circle implements Readable {
```
Comment (SofiaVynnytska12) [inline src/homework_5/ResizableCircle.java:7]: why do you implement Readable interface?

**Review by SofiaVynnytska12 — COMMENTED** [review]
Please, resolve merge conflicts

**Comment by Doctor-Ledenec** [comment]
I`ve fixed all problems and errors

**Comment by SofiaVynnytska12** [comment]
There are no more mistakes, but resolve merge conflicts, because I`m not able to merge the currect brunch now

**Comment by Doctor-Ledenec** [comment]
Merge conflicts have been solved

**Comment by SofiaVynnytska12** [comment]
and two commits ...

**Comment by Doctor-Ledenec** [comment]
I can't rollback a commit on GitHub so I see the only solution to this problem is to delete the branch and repost it to Github

**Comment by SofiaVynnytska12** [comment]
as you like, it is also a command git squash

**Pull request closed without merging**